bindings_ffi/gen_kotlin.sh

```diff
@@ -24,10 +24,11 @@ cargo run --bin ffi-uniffi-bindgen \
   --manifest-path $BINDINGS_MANIFEST \
   --features uniffi/cli --release -- \
   generate \
-  --lib-file $TARGET_DIR/release/lib$PROJECT_NAME.dylib $BINDINGS_PATH/src/$PROJECT_NAME.udl \
+  --lib-file $TARGET_DIR/release/lib$PROJECT_NAME.dylib $BINDINGS_PATH/LibXMTPKotlinFFI.zip \
```

Comment:
this command is failing to create the expected `xmtpv3.kt` android bindings, fails with an error `missing field package at line 161 column 1`

I was able to get it working again by re-adding the udl file that just has the namespace, see draft PR  here => 
https://github.com/xmtp/libxmtp/pull/1678/files

there's probably a better way to fix it, but we're unblocked for testing on android for now. maybe you have other ideas on how to fix. 

to validate, just need to make sure that xmtpv3.kt file is generated

Reply:
Thank you for getting to the bottom of that :pray: